source/DasBlog.Web.UI/Mappers/ProfileDasBlogUser.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AutoMapper;
+using DasBlog.Web.Core.Security;
+using DasBlog.Web.UI.Models.AccountViewModels;
+using DasBlog.Web.UI.Models.Identity;
+
+namespace DasBlog.Web.UI.Mappers
+{
+	public class ProfileDasBlogUser : Profile
+	{
+		public ProfileDasBlogUser()
+		{
+			CreateMap<RegisterViewModel, DasBlogUser>()
```

Comment:
Another great addition using mapper! @shanselman and I discussed this during our live hack